Lib/test/test_uuid.py

```diff
@@ -559,7 +571,10 @@ def test_ip_getnode(self):
 
     @unittest.skipUnless(os.name == 'posix', 'requires Posix')
```

Comment:
You should use @unittest.skipIf(AIX, "...").

Reply:
done